weather_mv/loader_pipeline/ee.py

```diff
@@ -294,8 +303,26 @@ def validate_arguments(cls, known_args: argparse.Namespace, pipeline_args: t.Lis
                             region=pipeline_options_dict.get('region'))
             logger.info('Region validation completed successfully.')
 
+        # Check for the band_names json file
+        if not os.path.exists(known_args.band_names):
+            raise RuntimeError("--band_names should contain a valid file that exists")
+        else:
+            if not known_args.band_names.split('.')[-1] == 'json':
```

Comment:
If I give a file with the filename - `imerg.band.names.json` here, it wouldn't work.

Can use `_, band_names_extension = os.path.splitext(known_args.band_names)`.